doc/security/security-overview.rst

```diff
@@ -150,8 +150,17 @@ The security architecture is based on a monolithic design where the
 Zephyr kernel and all applications are compiled into a single static
 binary. System calls are implemented as function calls without requiring
 context switches. Static linking eliminates the potential for
-dynamically loading malicious code. Memory protection and task
-separation techniques are in scope starting with version 1.10.0.
+dynamically loading malicious code.
+
+Additional protection features are available in later releases.  Stack
+protection mechanisms are provided to protect against stack overruns.
+In addition, applications can take advantage of thread separation
+features to split the system into privileged and privileged execution
```

Comment:
privileged and unprivileged

Reply:
heh, my fingers didn't type what my mind was thinking.  good catch.